frontend/app/package.json

```diff
@@ -79,9 +79,8 @@
     "@types/lodash": "4.14.195",
     "@types/node": "18.16.16",
     "@vitejs/plugin-vue2": "2.2.0",
-    "@vitest/coverage-c8": "0.31.4",
+    "@vitest/coverage-v8": "0.32.2",
     "@vue/test-utils": "1.3.5",
-    "c8": "7.14.0",
```

Comment:
OOT, is this needed?

Reply:
If things pass then we are safe right? 😄